### ISSUE TITLE
It's not possible to determine test coverage using NVDA

### ISSUE THREAD

**Issue by jeanbrazcosta** [microsoft/vscode#221647]
Type: <b>Feature Request</b>

When running test coverage, if there are lines in the code that haven't been covered, this information cannot be obtained using a screen reader.
To reproduce, you can use the attached project:
1. 
Ensure you have the Java Extension Pack installed.
2. 
When opening the Java project, navigate to the `CalculatorTest` class.
3. 
Open the code and press `Shift + F10` or right-click and choose `Run tests coverage in current file`.
4. 
In the command palette, type "Testing" and search for "focus testing coverage".
At this point, I attempted various methods to explore the results, including pressing Enter on the displayed methods, but it wasn't possible to determine whether a line had been covered by the test, the percentage of coverage, etc.
This attached project serves as an example, but you can use any other project.
Prerequisites:
• Java Extension Pack installed.

VS Code version: Code - Insiders 1.92.0-insider (a35380d6f0ff65f7c35345baafcf1c3f8a378d38, 2024-07-12T05:08:08.550Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Remote OS version: Linux x64 5.15.153.1-microsoft-standard-WSL2

**Comment by jeanbrazcosta** [comment]
[demo.zip](https://github.com/user-attachments/files/16203871/demo.zip)
Attached is the project I mentioned.

**Comment by connor4312** [comment]
Merging into https://github.com/microsoft/vscode/issues/209622
